vizro-core/src/vizro/static/css/bootstrap_overwrites.css

```diff
@@ -71,3 +71,9 @@ To ensure the dynamic height adjustment and prevent scrolling, the height must b
 label[for="theme-selector"] {
   display: none;
 }
+
+img[src*="icon-top"] {
+  filter: var(--fill-icon-image-card);
+  height: 36px;
+  width: 36px;
+}
```

Comment:
Have any of you been using this method for positioning an icon above a Card? It was supposed to be a short-cut method to have an icon switch colors with the theme.

If you frequently use it in demos and find it helpful, I would keep it here. Otherwise, I propose removing it entirely and adding it to our demos as custom CSS only and to our docs as an example.

@nadijagraca @antonymilne @petar-qb @maxschulz-COL @l0uden @Joseph-Perkins

Reply:
I haven't used it and the reason is that I haven't made many (if any) demo with an icon in a card 😄.

Reply:
I'm okay with removing it as well, I don't think I ever used it.

Reply:
I never used it so fine to remove 👍